test/datalog_tests/api_test/src/main.rs

```diff
@@ -0,0 +1,34 @@
+use copy_ddlog::api::*;
```

Comment:
This file also needs a comment indicating that it is a "main" entry point for a specific datalog program.